02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+const {
+    getPeople,
+    createPerson,
+    createPersonPostman,
+    updatePerson,
+    deletePerson,
+} = require("../controlers/people");
+
+//1st way
+// //get method HTTP
+// router.get('/',getPeople)
+// router.post('/',createPerson);
+// //postman
+// router.post("/postman",createPersonPostman);    
+// //Put Method HTTP
+// router.put("/:id",updatePerson);    
+// //Delete Method HTTP
+// router.delete("/:id",deletePerson);
+
+//2nd way
+router.route('/').get(getPeople).post(createPerson);
+router.route('/postman').post(createPersonPostman);
+router.route('/:id').put(updatePerson).delete(deletePerson);
```

Comment:
Nitpick (doesn't require changes):
When naming endpoints, I like to prefix them with an action, so it's easier to read/understand what the route expects to happen. 

For example:
```js
// instead of...
// http://localhost:3000/api/people/6
router.delete('/:id', deletePerson);

// it becomes...
// http://localhost:3000/api/people/delete/6
router.delete('/delete/:id', deletePerson);
```

Reply:
Thank you so much Matt. Thank you for your patience in navigating through my errors Your ability to provide solutions with such clarity and understanding has been a constant source of inspiration for me. I am incredibly fortunate to have the opportunity to learn from someone as experienced and knowledgeable as you.